client/gutenberg/extensions/publicize/store/selectors.js

```diff
@@ -6,5 +6,18 @@
  * @return {Array} List of connections.
  */
 export function getFailedConnections( state ) {
-	return state.filter( connection => ! connection.test_success );
+	return state.filter( connection => false === connection.test_success );
+}
+
+/**
+ * Returns Publicize connections that require reauthentication from users. For example, when LinkedIn switched its API from v1 to v2.
```

Comment:
I find this a bit misleading - it doesn't actually return connections; it rather returns connection service names. Let's make sure to reflect that in the docs.

Reply:
Ok will update